runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -183,6 +191,17 @@ public void scheduledChecks()
 				}
 			}
 		}
+
+		if (getInfoTimer() != null)
+		{
+			Duration statTimeout = Duration.ofMinutes(config.statTimeout());
```

Comment:
Just add a field at the top that contains the `statTimeout`, update it on `startup` and on `configChanged`, no need to create the object each time you calculate the elapsed time.